tests/test_vbox.py

```diff
@@ -20,3 +20,22 @@ def test_vbox():
     with StringIO(xml.strip()) as fh:
         vbox = VBox(fh)
         assert next(vbox.disks()) == "os2warp4.vdi"
+
+
+def test_vbox_lowercase_disk_format():
```

Comment:
For reviewer: Eventually we can parameterize this, but since it is only two cases now this seems clearer.